lib/core/batchingReceiver.ts

```diff
@@ -55,6 +62,10 @@ export class BatchingReceiver extends MessageReceiver {
       maxWaitTimeInSeconds = Constants.defaultOperationTimeoutInSeconds;
     }
 
+    if (maxMessageWaitTimeoutInSeconds == undefined) {
+      maxMessageWaitTimeoutInSeconds = 2;
```

Comment:
I tried 0.1 seconds but it didn't work well. It was too fast and ends up stopping the receiver before starting it. I tried an example with 2 seconds and it worked pretty well. Will stick to this for now. Feel free to change it later based on your findings.